export/pyproject.toml

```diff
@@ -11,13 +11,23 @@ python = "^3.11"
 pexpect = "^4.9.0"
 
 [tool.poetry.group.dev.dependencies]
+# In production these two are installed using a system package
```

Comment:
Because of the complications around compiling these, I think we should drop these from poetry (maybe leave a comment though). For CI any extra dependencies that are needed can be added to the existing `ci-install-deps` target in the Makefile.